README.md

```diff
@@ -1,74 +1,270 @@
+   
+   Recycles the current (the one you initiated last) Flex Service. To recycle a different service, specify its service ID using the `--serviceId` option.
+
+`flex delete`
+
+   Deletes the current Flex Service configuration from the Node.js project directory that it has been executed in.
+
+`help`
+
+   Prints general usage instructions. For detailed command usage instruction, use the `--help` option with the command.
+
+## Options
+
+`--domain <app|org>`
+
+   Specifies the domain type for commands like `flex list`.
+
+`--email <e-mail>`
+
+   Email address of your Kinvey account.
+
+`--host <host>`
+
+   Hostname of a Kinvey instance. It has a default value of `https://manage.kinvey.com/` which most customers should use. If you are a customer on a dedicated Kinvey instance, enter your dedicated host name as either an absolute URI (`https://kvy-us2-manage.kinvey.com/`) or an instance name only (`kvy-us2`).
+
+`--id <app or organization ID>`
+
+   App or organization ID for use with `--domain <app|org>`.
+
+`--password <password>`
+
+   Password for your Kinvey account.
+
+`--profile <profile>`
+
+   Profile to use.
+
+`--serviceId <Flex Service ID>`
+
+   Specifies a Flex Service, e.g. for use with the `flex status` command.
+
+`--silent`
+
+   Suppresses any output. Useful for scripting.
+
+`--suppress-version-check`
+
+   Prevents Kinvey CLI to check for new versions, which normally happens each execution.
+   
+`--verbose`
+
+   Prints additional debug messages.
+
+`--version`
+
+   Prints the version number of `kinvey-cli`.
+
+`--help, -h`
+
+   When used after a `kinvey-cli` command, shows its usage instructions.
+
+## Environment Variables
+
+Use these environment variables to specify profile information for the `profile create` command when you don't want to specify it on the command line. Keep in mind that any values specified at he command line take precedence over the environment variable values.
+
+`KINVEY_CLI_EMAIL`
+
+   Email address of your Kinvey account.
+
+`KINVEY_CLI_PASSWORD`
+
+   Password for your Kinvey account.
+
+`KINVEY_CLI_HOST`
+
+   Hostname of a Kinvey instance. It has a default value of `https://manage.kinvey.com/` which most customers should use. If you are a customer on a dedicated Kinvey instance, enter your dedicated host name as either an absolute URI (`https://kvy-us2-manage.kinvey.com/`) or an instance name only (`kvy-us2`).
+
+`KINVEY_CLI_PROFILE`
+
+   Profile to use.
+
+
+## Getting Help
+
+Kinvey CLI comes with a two-stage help system. You can either call the `help` command to see an overview of the available commands or request details about a command usage with the `-h` flag.
+
+    kinvey help
+
+    kinvey flex -h
+
+    kinvey flex logs -h
+
+
+## Getting Started
+
+Kinvey CLI requires you to authenticate and to have a working profile. The fastest way to get started is to run the `kinvey init` command. It prompts for credentials and hostname and creates a working profile for you.
```

Comment:
"Kinvey CLI requires you to authenticate and to have a working profile. " - you can use the cli and deploy a service even without a profile

Reply:
👍